conans/client/conan_api.py

```diff
@@ -384,41 +384,48 @@ def export_pkg(self, conanfile_path, name, channel, source_folder=None, build_fo
         env = env or []
         cwd = cwd or get_cwd()
 
-        # Checks that info files exists if the install folder is specified
-        if install_folder and not existing_info_files(_make_abs_path(install_folder, cwd)):
-            raise ConanException("The specified install folder doesn't contain '%s' and '%s' "
-                                 "files" % (CONANINFO, BUILD_INFO))
+        try:
+            recorder = ActionRecorder()
```

Comment:
I think this pattern is a bit ugly and repetitive, sounds like we should try to improve it, maybe embedding it in the decorator or something. Not for this PR, just an idea.

Reply:
Yes, it will be useful. I'll open an issue with this task